crates/aptos-crypto/src/traits.rs

```diff
@@ -310,4 +310,8 @@ pub(crate) mod private {
     impl Sealed for crate::multi_ed25519::MultiEd25519PrivateKey {}
     impl Sealed for crate::multi_ed25519::MultiEd25519PublicKey {}
     impl Sealed for crate::multi_ed25519::MultiEd25519Signature {}
+
+    impl Sealed for crate::bls12381::BLS12381PrivateKey {}
+    impl Sealed for crate::bls12381::BLS12381PublicKey {}
+    impl Sealed for crate::bls12381::BLS12381Signature {}
```

Comment:
I kinda like the model employed by our x25519 module, where we don't have ridiculously long-named types. Use bls12381::Signature etc.

Reply:
We would need to avoid a name clash with the Signature interface... that module actually does not have a Signature struct AFAICT.

Reply:
Actually, even for PublicKey and PrivateKey there are traits with the same name.

If we're gonna figure out a way to do this, it might be worth noting that the Ed25519 module uses `Ed25519` prefixes, unlike the x25519 module.

Reply:
Resolved.